feather/protocol/src/packets/server/play.rs

```diff
@@ -1179,6 +1181,163 @@ packets! {
+            "generic:knockback_resistance" => EntityAttributeKind::KnockbackResistance,
+            "generic:movement_speed" => EntityAttributeKind::MovementSpeed,
+            "generic:attack_damage" => EntityAttributeKind::AttackDamage,
+            "generic:attack_speed" => EntityAttributeKind::AttackSpeed,
+            "generic:attack_knockback" => EntityAttributeKind::AttackKnockback,
+            "generic:flying_speed" => EntityAttributeKind::FlyingSpeed,
+            "generic:armor" => EntityAttributeKind::Armor,
+            "generic:armor_toughness" => EntityAttributeKind::ArmorToughness,
+            "generic:luck" => EntityAttributeKind::Luck,
+            "horse:jump_strength" => EntityAttributeKind::HorseJumpStrength,
+            "zombie:spawn_reinforcements" => EntityAttributeKind::ZombieSpawnReinforcements,
+            _ => return Err(anyhow::anyhow!("invalid entity property key: '{}'", key)),
+        })
+    }
+
+    pub fn id(&self) -> String {
+        String::from(match self {
+            EntityAttributeKind::MaxHealth => "generic:max_health",
+            EntityAttributeKind::FollowRange => "generic:follow_range",
+            EntityAttributeKind::KnockbackResistance => "generic:knockback_resistance",
+            EntityAttributeKind::MovementSpeed => "generic:movement_speed",
+            EntityAttributeKind::AttackDamage => "generic:attack_damage",
+            EntityAttributeKind::AttackSpeed => "generic:attack_speed",
+            EntityAttributeKind::AttackKnockback => "generic:attack_knockback",
+            EntityAttributeKind::FlyingSpeed => "generic:flying_speed",
+            EntityAttributeKind::Armor => "generic:armor",
+            EntityAttributeKind::ArmorToughness => "generic:armor_toughness",
+            EntityAttributeKind::Luck => "generic:luck",
+            EntityAttributeKind::HorseJumpStrength => "horse:jump_strength",
+            EntityAttributeKind::ZombieSpawnReinforcements => "zombie:spawn_reinforcements",
+        })
+    }
+
+    pub fn max_value(&self) -> f32 {
+        match &self {
+            EntityAttributeKind::MaxHealth => 1024.0,
+            EntityAttributeKind::FollowRange => 2048.0,
+            EntityAttributeKind::KnockbackResistance => 1.0,
+            EntityAttributeKind::MovementSpeed => 1024.0,
+            EntityAttributeKind::AttackDamage => 2048.0,
+            EntityAttributeKind::AttackSpeed => 1024.0,
+            EntityAttributeKind::AttackKnockback => 5.0,
+            EntityAttributeKind::FlyingSpeed => 1024.0,
+            EntityAttributeKind::Armor => 30.0,
+            EntityAttributeKind::ArmorToughness => 20.0,
+            EntityAttributeKind::Luck => 1024.0,
+            EntityAttributeKind::HorseJumpStrength => 2.0,
+            EntityAttributeKind::ZombieSpawnReinforcements => 1.0,
+        }
+    }
+
+    pub fn min_value(&self) -> f32 {
+        if let EntityAttributeKind::Luck = &self {
```

Comment:
I would actually rather make this a match as well because it makes more semantic sense imo.
```rust
match &self {
    EntityAttributeKind::Luck => -1024.0,
    _ => 0.0,
}
```

Reply:
I think clippy complains about that.